tests/unit/pages/tutor-profile/TutorProfile.spec.jsx

```diff
@@ -76,12 +84,60 @@ describe('TutorProfile', () => {
 
     useAxios.mockImplementation(() => fakeData)
     renderWithProviders(<TutorProfile />, {
-      initialEntries: route,
+      initialEntries: tutorRoute,
       preloadedState: { appMain }
     })
 
     const name = screen.getByText(`${mockData.firstName} ${mockData.lastName}`)
 
     expect(name).toBeInTheDocument()
   })
+
+  it('Should render video presentation block for tutor', () => {
+    const fakeData = getFakeData(false)
+
+    useAxios.mockImplementation(() => fakeData)
+    renderWithProviders(<TutorProfile />, {
+      initialEntries: tutorRoute,
+      preloadedState: { appMain }
+    })
+
+    const videoBlockTitle = screen.getByText(
+      'tutorProfilePage.videoPresentation.title'
+    )
+
+    expect(videoBlockTitle).toBeInTheDocument()
+  })
+
+  it('Should not render video presentation block when student has no video link', () => {
+    const fakeData = getFakeData(false)
+
+    useAxios.mockImplementation(() => fakeData)
+    renderWithProviders(<TutorProfile />, {
+      initialEntries: studentRoute,
+      preloadedState: { appMain }
```

Comment:
In this test you test behavior, when role is student, but use the same preloadedState (tutor)

Reply:
User role for tests was taken from `studentRoute` and `tutorRoute` variables. It worked, as inside of `TutorProfile` there is such condition: `paramsRole || userRole` (paramsRole is from search params, userRole is from redux appMain). But it really can be confusing, so in tests, I removed roles from routes and instead set them in appMain.